README.rst

```diff
@@ -78,6 +78,33 @@ You have the choice for 2 different textual formats that will be sent to the kin
             "pkey": <...>
           }
 
+* ``JSONPayloadFormatter``: handles changes decoded by the wal2json plugin and outputs either the primary key of the change or the full changed row.::
```

Comment:
Is there a reason the below formats can't be unified so they both are lists? i.e.
A list of 1 column or a list of many columns vs a scalar of 1 column or a list of many.